ini/stm32f4.ini

```diff
@@ -589,3 +589,27 @@ build_flags       = ${common_stm32.build_flags}
                     -DFLASH_DATA_SECTOR=1U -DFLASH_BASE_ADDRESS=0x08004000
 extra_scripts     = ${common_stm32.extra_scripts}
   pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
+
+#
+# Ender 3S1 STM32F401RC_creality
+#
+[env:STM32F401RC_creality]
+extends                     = stm32_variant
+board                       = genericSTM32F401RC
+board_build.variant         = MARLIN_CREALITY_STM32F401RC
+board_build.offset          = 0x10000
+board_upload.offset_address = 0x08007000
+build_flags                 = ${stm32_variant.build_flags}
+                              -DSS_TIMER=4 -DTIMER_SERVO=TIM5
+                              -DENABLE_HWSERIAL3 -DTRANSFER_CLOCK_DIV=8
+                              -DMCU_STM32F401RC -DSTM32F4
+                              -lproui -LMarlin/lib/proui
```

Comment:
Was this line supposed to be kept for upstream, or is it specific to youre repo?

Reply:
Oh! I forgot to delete it.